c-sharp-chat/PubnubChatApi/PubnubChatApi/Entities/Chat.cs

```diff
@@ -711,8 +715,12 @@ public async Task<Channel> CreatePublicConversation(string channelId, ChatChanne
             return channel;
         }
 
-        public async Task<CreatedChannelWrapper> CreateDirectConversation(User user, string channelId)
+        public async Task<CreatedChannelWrapper> CreateDirectConversation(User user, string channelId = "")
```

Comment:
Shouldn't you mention all those small adjustments also in the description (targeting change notes) ?